modules/transactions.js

```diff
@@ -381,18 +432,41 @@ Transactions.prototype.onPeersReady = function () {
 
 // Shared
 shared.getTransactions = function (req, cb) {
-	library.schema.validate(req.body, schema.getTransactions, function (err) {
-		if (err) {
-			return setImmediate(cb, err[0].message);
-		}
-
-		__private.list(req.body, function (err, data) {
-			if (err) {
-				return setImmediate(cb, 'Failed to get transactions: ' + err);
-			}
+	async.waterfall([
+		function (waterCb) {
+			var params = {},
+				pattern = /(and|or){1}:/i;
+
+			// Filter out 'and:'/'or:' from params to perform schema validation
+			_.each (req.body, function (value, key) {
```

Comment:
Please remove space between function name and arguments list. Without space equals invocation, with space equals declaration. Also see other occurrences within this PR.